minigotchi-ESP32/display.h

```diff
@@ -1,29 +1,33 @@
-/**
- * display.h: header files for display.cpp
- */
-
 #ifndef DISPLAY_H
 #define DISPLAY_H
 
-#include "config.h"
-#include <Adafruit_GFX.h>
 #include <Adafruit_SSD1306.h>
-#include <SPI.h>
 #include <Wire.h>
+#include <TFT_eSPI.h>  // Defines the TFT_eSPI library for CYD
+#include "Config.h"
```

Comment:
i think it's just `config.h` btw

Reply:
Are you referring to the capitalization of Config?

Reply:
Edited for integrity